### ISSUE TITLE
Using ALTER TABLE to add an index after INSERT dumps the core

### ISSUE THREAD

**Issue by klassiker** [edgelesssys/edgelessdb#93]
I followed https://docs.edgeless.systems/edgelessdb/#/getting-started/quickstart-sgx to set up a simple container. After successfully connecting, the following SQL statements will cause a coredump:

```sql
CREATE DATABASE test;

USE test;

CREATE TABLE `test` (
    `test` int
);

INSERT INTO `test` (`test`) VALUES (1);

ALTER TABLE `test` ADD INDEX `idx_test` (`test`);
```

Without the `INSERT`, everything works fine.

**Comment by thomasten** [comment]
Thanks for reporting! We can reproduce the bug and will investigate.

**Comment by thomasten** [comment]
This has been fixed in main branch and will be part of the next release.

**Comment by klassiker** [comment]
Thank you!

**Comment by thomasten** [comment]
fixed in v0.3.0

**Comment by klassiker** [comment]
Works for me.
